tests/snapshot-gating.fmf

```diff
@@ -23,6 +23,11 @@ prepare:
     script: |
       dnf install -y dnf-plugins-core
       dnf config-manager --save --setopt="testing-farm-tag-repository.priority=999" || true
+
+  - name: Assert correct llvm package is installed
+    how: shell
+    order: 99
+    script: tests/assert_llvm.sh
```

Comment:
```suggestion
    script: rpm -qi llvm-libs | grep -P "Version.*~pre.*"
```

This should do the trick.

Reply:
I understand that your point here is to keep it simple. I do think that the script might be more beneficial in this case, though. The script gives us some advantages here:

- We can make several checks at the `prepare` step end, all grouped under the same script. This may include, checking that the copr repo is reachable.
- We can write more complex logic. For example testing several packages rather that llvm-libs only.
- We can properly report what failed and why.

For example, upon wrong package installation with the script, this is what we see in the report:
```
    prepare step failed

    The exception was caused by 1 earlier exceptions

    Cause number 1:

        Command '/var/ARTIFACTS/work-snapshot-gatingejv1c0ks/tests/snapshot-gating/tree/tmt-prepare-wrapper.sh-Assert-correct-llvm-package-is-installed-default-0' returned 1.

        stdout (2 lines)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        *** Unexpected llvm package installed: llvm-18.1.8-2.fc41.x86_64
        *** Expected llvm: llvm-20.0.0~pre20240825.g3ef64f7ab5b865-1.fc42.x86_64
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
However, using the plain command, we would see this instead, which is much less intuitive:

```
    prepare step failed

    The exception was caused by 1 earlier exceptions

    Cause number 1:

        Command '/var/tmp/tmt/run-077/tests/snapshot-gating/tree/tmt-prepare-wrapper.sh-Assert-correct-llvm-package-is-installed-default-0' returned 1.

```